### DISCUSSION TITLE
数据库小问题

### DISCUSSION THREAD

**Discussion by Nofated095** [walinejs/waline#748] [Q&A] [answered]
![Screenshot_20220109-155131](https://user-images.githubusercontent.com/49985975/148673920-261f9fb3-8a61-4f75-9c6c-a5bb272ad010.png)
如图所示，在一个页面中，如果浏览器中地址不一样，虽然加载出来的页面一样，但因为最后地址有或者没有 / 就导致评论加载不出来

![Screenshot_20220109-155430](https://user-images.githubusercontent.com/49985975/148674006-7c990d32-8069-4ffa-a547-a31ccda5575c.png)
![Screenshot_20220109-155443](https://user-images.githubusercontent.com/49985975/148674014-4e2d624b-6925-4192-a7e4-8e2bf246c364.png)

**Comment by lizheming** [comment, marked as answer]
Waline 默认使用的是路由作为评论标识符，但他的本质实际上就是文章和评论之前的对应关系，你们可以根据自己情况去做自定义。比如你现在的情况你是可以自己增加 `/` 标识来处理的。

```
Waline({
  path: location.pathname + !location.pathname.endsWith('/') ? '/' : ''
})
```

**Comment by Mister-Hope** [comment]
我再解释一句，对于有些人的网站来说 a对应a.html 而 a/ 对应 a/index.html 因此它们可以对于某些人是完全不同的地址，所以请你自己处理类似的问题。
